### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by hgvanpariya**
pull request template #21

[
screen shot 2018-04-21 at 2 01 05 pm
](url)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+### Pull Request check-list
```
[the remark this answers is not in the document]
Comment (boyeborg) [reply .github/pull_request_template.md]: I would suggest a comment like:

> Unless this is a minor change (like fixing a typo), remember to create an issue before submitting a pull request. This way you can get feedback from others before you start working on the issue. Remember to reference the issue in the body of your pull request.

on the top of the template using the HTML comment tags, like: `<!-- COMMENT -->`

**Comment by hgvanpariya** [comment]
@boyeborg , @siddise, I've updated the template, please provide your feedback ...

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+<!-- Unless this is a minor change (like fixing a typo), remember to create an issue before submitting a pull request. This way you can get feedback from others before you start working on the issue. Remember to reference the issue in the body of your pull request. -->
+
+_Please make sure to review and check all of these items:_
```
Comment (siddise) [inline .github/pull_request_template.md]: Probably won't need this line since there is only one checkbox

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+### Pull Request check-list
```
Comment (siddise) [inline .github/pull_request_template.md]: Remove this

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<!-- Unless this is a minor change (like fixing a typo), remember to create an issue before submitting a pull request. This way you can get feedback from others before you start working on the issue. Remember to reference the issue in the body of your pull request. -->
+
+_Please make sure to review and check all of these items:_
+
+- [ ] Have you followed the contributing guidelines? 
+
+IMPORTANT: Please review the [CONTRIBUTING.md](../CONTRIBUTING.md) file for detailed contributing guidelines.
+ 
+### Affected Workflow
```
Comment (siddise) [inline .github/pull_request_template.md]: Remove this

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+<!-- Please provide affected Workflow -->
+
+### Desired to formulate issue
```
Comment (siddise) [inline .github/pull_request_template.md]: Remove this

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+<!-- Please provide desired to formulate issue. -->
+
+### Commit message
```
Comment (siddise) [inline .github/pull_request_template.md]: Remove this

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+<!-- Unless this is a minor change (like fixing a typo), remember to create an issue before submitting a pull request. This way you can get feedback from others before you start working on the issue. Remember to reference the issue in the body of your pull request. -->
+
+_Please make sure to review and check all of these items:_
+
+- [ ] Have you followed the contributing guidelines? 
+
+IMPORTANT: Please review the [CONTRIBUTING.md](../CONTRIBUTING.md) file for detailed contributing guidelines.
+ 
+### Affected Workflow
+<!-- Please provide affected Workflow -->
```
Comment (siddise) [inline .github/pull_request_template.md]: Edit this to say

File: .github/pull_request_template.md (hunk 4 above)
Comment (hgvanpariya) [reply .github/pull_request_template.md]: ### Affected Workflow Description ?

**Comment by hgvanpariya** [comment]
@siddise : please review ...

**Comment by siddise** [comment]
@hgvanpariya Could you rename your commit message to "add pull request template"?

**Comment by hgvanpariya** [comment]
@siddise , updated the PR comment ...

**Comment by hgvanpariya** [comment]
@siddise , Thank you ...

**Pull request merged**